packages/tendermint-rpc/src/v0-33/responses.ts

```diff
@@ -62,7 +82,7 @@ function decodeAbciQuery(data: RpcAbciQueryResponse): responses.AbciQueryRespons
   return {
     key: Base64.decode(optional(data.key, "" as Base64String)),
     value: Base64.decode(optional(data.value, "" as Base64String)),
-    // proof: may(Base64.decode, data.proof),
+    proof: may(decodeQueryProof, data.proof),
```

Comment:
I think we should test this field in a plain Tendermint test. Is this possible with our testing `--proxy_app=kvstore`?

Reply:
There are no proofs there. We can get undefined.

The actual format we parse is only from the cosmos-sdk